src/main/java/com/minecolonies/commands/AbstractSingleCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.minecolonies.commands;
+
+import net.minecraft.command.ICommandSender;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A command that has children.
```

Comment:
Same javadoc as AbstractSplitCommand, but they seem different. I think this should be expanded.